### PR TITLE
sosreport.py:Removed ticket-number and -z bzip2 options

### PR DESCRIPTION
**Pull request by shirishaganta1**
The options ticket-number and -z bzip2 are not supporting

Signed-off-by: Shirisha Ganta <SHIRISHA.Ganta1@ibm.com>

### CODE REVIEW

**Comment by shirishaganta1** [comment]
Before Patch:
[root@ltc-zz189-lp1 ras]# avocado run sosreport.py
JOB ID     : 4547309889fdc222435fc1281051a753001939de
JOB LOG    : /home/siri/avocado-fvt-wrapper/results/job-2021-01-12T03.33-4547309/job.log
 (1/6) sosreport.py:Sosreport.test_short: FAIL: 1 command(s) failed in sosreport tool verification (214.34 s)
 (2/6) sosreport.py:Sosreport.test_user: PASS (65.14 s)
 (3/6) sosreport.py:Sosreport.test_plugins: FAIL: 1 command(s) failed in sosreport tool verification (189.41 s)
 (4/6) sosreport.py:Sosreport.test_others: PASS (313.16 s)
 (5/6) sosreport.py:Sosreport.test_archive: FAIL: 1 command(s) failed in sosreport tool verification (191.75 s)
 (6/6) sosreport.py:Sosreport.test_PPC: PASS (9.28 s)
RESULTS    : PASS 3 | ERROR 0 | FAIL 3 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/siri/avocado-fvt-wrapper/results/job-2021-01-12T03.33-4547309/results.html
JOB TIME   : 984.55 s
[root@ltc-zz189-lp1 ras]# 

After Patch:
[root@ltc-zz189-lp1 ras]# avocado run sosreport.py
JOB ID     : eb3f581d6bcc3a3de1d6d32a911c1f188e083fbc
JOB LOG    : /home/siri/avocado-fvt-wrapper/results/job-2021-01-12T06.39-eb3f581/job.log
 (1/6) sosreport.py:Sosreport.test_short: PASS (194.88 s)
 (2/6) sosreport.py:Sosreport.test_user: PASS (66.95 s)
 (3/6) sosreport.py:Sosreport.test_plugins: FAIL: 1 command(s) failed in sosreport tool verification (193.71 s)
 (4/6) sosreport.py:Sosreport.test_others: PASS (313.37 s)
 (5/6) sosreport.py:Sosreport.test_archive: PASS (190.73 s)
 (6/6) sosreport.py:Sosreport.test_PPC: PASS (9.80 s)
RESULTS    : PASS 5 | ERROR 0 | FAIL 1 | SKIP 0 | WARN 0 | INTERRUPT 0 | CANCEL 0
JOB HTML   : /home/siri/avocado-fvt-wrapper/results/job-2021-01-12T06.39-eb3f581/results.html
JOB TIME   : 970.69 s
[root@ltc-zz189-lp1 ras]#

File: ras/sosreport.py [hunk 1]
```diff
@@ -75,13 +74,6 @@ def test_short(self):
         self.run_cmd("sosreport -h", False)
         self.run_cmd("sosreport -l", False)
         self.run_cmd("sosreport --batch --tmp-dir=%s --verify" % directory_name)
-
-        ticket_id = self.params.get('ticket_id', default='testid')
```
Comment (harish-24) [inline ras/sosreport.py:79]: Can you also remove the same from the yaml file?

Comment (shirishaganta1) [reply ras/sosreport.py:79]: sure

**Review by harish-24 — APPROVED** [review]
LGTM

**Pull request merged**